Web/Edubase.Web.UI/Areas/Groups/Views/Group/EditDetails.cshtml

```diff
@@ -206,7 +210,7 @@
                     }
 
                     @* CLOSED DATE *@
-                    @if (!Model.GroupType.OneOfThese(L.MultiacademyTrust, L.SingleacademyTrust) || Model.CanUserEditClosedDate)
+                    @if (!Model.GroupType.OneOfThese(L.MultiacademyTrust, L.SingleacademyTrust, L.SecureSingleAcademyTrust) || Model.CanUserEditClosedDate)
                     {
```

Comment:
Will this enable YCS to edit close date? Spreadsheet says they're not allowed to.

Reply:
see this on the AC's for YCS edit:

GIVEN I'm logged into GIAS as a YCS user
WHEN searching for a Secure Single-academy trust group
AND editing an group
AND setting the Status to Closed
THEN the read-only Closed date field becomes editable

Reply:
Waiting for response on this from Mark